internal/util/gitlab/gitlab.go

```diff
@@ -60,6 +62,12 @@ type bodyReq struct {
 	PrivateToken string `json:"private_token"`
 }
 
+type gitlabMR struct {
+	ID           int    `json:"id"`
+	IID          int    `json:"iid"`
```

Comment:
Can we add comments like what's ID and IID? Or maybe adding the ref URL

Reply:
Okay, i'll add a reference url